### ISSUE TITLE
Fix focusing on text editor (contenteditable="true")

### ISSUE THREAD

**Issue by CoverID** [OnsenUI/OnsenUI#1306]
Hello guys, I created an app with Summernote as my text editor and I cant focus on my text editor (tested on IOS), then I searching on google and found this script to fix my problem
[https://github.com/ftlabs/fastclick/pull/211/commits/ad47104b15eb95b5fdf2f1c5df9415045aaf5a58](https://github.com/ftlabs/fastclick/pull/211/commits/ad47104b15eb95b5fdf2f1c5df9415045aaf5a58)
and I added css for my text editor like this :
`.textEditorClass {
-webkit-user-select: text !important;
-moz-user-select: text !important;
-ms-user-select: text !important;
user-select: text !important;
}`
I dont know if its work too if I put :not(.textEditorClass) at *:not(input):not(textarea):not(select) in onsen-css-components.css #

**Comment by argelius** [comment]
I'm not familiar with Summernote so unfortunately I can't help you.
If you figure out how to fix it please open a PR.

**Comment by CoverID** [comment]
I use onsenui - v1.3.15 - 2016-01-19
I figure out how to fix it for but im new with github, maybe need more time to learn
I've checked this version, if still using `fastclick-patched.js` just add few script after line 400
[https://github.com/OnsenUI/OnsenUI/blob/master/core/vendor/fastclick-patched.js#L400](https://github.com/OnsenUI/OnsenUI/blob/master/core/vendor/fastclick-patched.js#L400)

```
if (targetElement.isContentEditable) {
    return true;
}
```
and add this to your css if necessary
```
[contenteditable=true] {
    -webkit-user-select: text;
    -moz-user-select: text;
    -ms-user-select: text;
    user-select: text;
}
```
or put [contenteditable=true] together with input and textarea rule

**Comment by asial-matagawa** [comment]
@CoverID 
This problem solved in #1646. I will close this issue.
